{
	"dataType": "CVE_RECORD",
	"dataVersion": "5.1",
	"cveMetadata": {
		"cveId": "CVE-2021-33571",
		"assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
		"assignerShortName": "mitre",
		"state": "PUBLISHED",
		"datePublished": "2021-06-08T00:00:00",
		"dateUpdated": "2024-08-03T23:50:43.143Z",
		"dateReserved": "2021-05-25T00:00:00"
	},
	"containers": {
		"cna": {
			"providerMetadata": {
				"orgID": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
				"shortName": "mitre",
				"dateUpdated": "2023-12-07T22:09:18.547259"
			},
			"descriptions": [
				{
					"lang": "en",
					"value": "In Django 2.2 before 2.2.24, 3.x before 3.1.12, and 3.2 before 3.2.4, URLValidator, validate_ipv4_address, and validate_ipv46_address do not prohibit leading zero characters in octal literals. This may allow a bypass of access control that is based on IP addresses. (validate_ipv4_address and validate_ipv46_address are unaffected with Python 3.9.5+..) ."
				}
			],
			"affected": [
				{
					"vendor": "n/a",
					"product": "n/a",
					"versions": [
						{
							"status": "affected",
							"version": "n/a"
						}
					]
				}
			],
			"problemTypes": [
				{
					"descriptions": [
						{
							"type": "text",
							"lang": "en",
							"description": "n/a"
						}
					]
				}
			],
			"references": [
				{
					"url": "https://docs.djangoproject.com/en/3.2/releases/security/"
				},
				{
					"url": "https://groups.google.com/g/django-announce/c/sPyjSKMi8Eo"
				},
				{
					"url": "https://www.djangoproject.com/weblog/2021/jun/02/security-releases/"
				},
				{
					"url": "https://security.netapp.com/advisory/ntap-20210727-0004/"
				},
				{
					"name": "FEDORA-2022-e7fd530688",
					"tags": [
						"vendor-advisory"
					],
					"url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/B4SQG2EAF4WCI2SLRL6XRDJ3RPK3ZRDV/"
				},
				{
					"url": "https://github.com/django/django/commit/f27c38ab5d90f68c9dd60cabef248a570c0be8fc"
				},
				{
					"url": "https://github.com/django/django/commit/203d4ab9ebcd72fc4d6eb7398e66ed9e474e118e"
				},
				{
					"url": "https://github.com/django/django/commit/9f75e2e562fa0c0482f3dde6fc7399a9070b4a3d"
				}
			]
		},
		"adp": [
			{
				"providerMetadata": {
					"orgID": "af854a3a-2127-422b-91ae-364da2661108",
					"shortName": "CVE",
					"dateUpdated": "2024-08-03T23:50:43.143Z"
				},
				"title": "CVE Program Container",
				"references": [
					{
						"tags": [
							"x_transferred"
						],
						"url": "https://docs.djangoproject.com/en/3.2/releases/security/"
					},
					{
						"tags": [
							"x_transferred"
						],
						"url": "https://groups.google.com/g/django-announce/c/sPyjSKMi8Eo"
					},
					{
						"tags": [
							"x_transferred"
						],
						"url": "https://www.djangoproject.com/weblog/2021/jun/02/security-releases/"
					},
					{
						"tags": [
							"x_transferred"
						],
						"url": "https://security.netapp.com/advisory/ntap-20210727-0004/"
					},
					{
						"name": "FEDORA-2022-e7fd530688",
						"tags": [
							"vendor-advisory",
							"x_transferred"
						],
						"url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/B4SQG2EAF4WCI2SLRL6XRDJ3RPK3ZRDV/"
					},
					{
						"tags": [
							"x_transferred"
						],
						"url": "https://github.com/django/django/commit/f27c38ab5d90f68c9dd60cabef248a570c0be8fc"
					},
					{
						"tags": [
							"x_transferred"
						],
						"url": "https://github.com/django/django/commit/203d4ab9ebcd72fc4d6eb7398e66ed9e474e118e"
					},
					{
						"tags": [
							"x_transferred"
						],
						"url": "https://github.com/django/django/commit/9f75e2e562fa0c0482f3dde6fc7399a9070b4a3d"
					}
				]
			}
		]
	}
}
